github/filecoin-project.yml

```diff
@@ -3457,8 +3463,8 @@ repositories:
     has_discussions: false
     merge_commit_message: PR_TITLE
     merge_commit_title: MERGE_MESSAGE
-    secret_scanning_push_protection: false
-    secret_scanning: false
+    secret_scanning_push_protection: true
```

Comment:
what does this flag do and why changes it in this PR?

Reply:
This is related to state being out of sync.  I've asked in the IPDX/FilOz channel the right way to clear this up: https://filecoinproject.slack.com/archives/C06CY12V83S/p1726674153573169

I'm also running the sync workflow to see if that clears things up: https://github.com/filecoin-project/github-mgmt/actions/runs/10925569479

We shouldn't merge until the terraform plan is only touching resources we'd expect in the way we'd expect

Reply:
Once I get the state sync figured out (i.e., no unrelated changes) are you good with merging @jennijuju ?